frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -1555,11 +1570,14 @@ def add(self, resulting_type: ConversionType, x: Conversion, y: Conversion) -> C
         if x.is_clear and y.is_clear:
             highlights = {
                 x.origin: "lhs is clear",
-                y.origin: "rhs is clear" if x.origin is not y.origin else "operand is clear",
+                y.origin: ("rhs is clear" if x.origin is not y.origin else "operand is clear"),
                 self.converting: "but clear-clear additions are not supported",
             }
             self.error(highlights)
 
+        print("resulting type", resulting_type.mlir)
+        print("x type", x.type.mlir)
+        print("y type", y.type.mlir)
```

Comment:
I guess these are temporary, but let's not forget to delete them when the time comes.